webapp/src/assets/config.dedis.toml

```diff
@@ -1,4 +1,5 @@
 ByzCoinID = "9cc36071ccb902a1de7e0d21a2c176d73894b1cf88ae4cc2ba4c95cd76f474f3"
+SignupNode = "https://byzcoin-signup.c4dt.org"
```

Comment:
not really related to the PR; `curl http://byzcoin-signup.c4dt.org` (via http not https) gives me a 404, shouldn't it redirect to the https version instead?